### ISSUE TITLE
Feature request: Update a11y baseline

### ISSUE THREAD

**Issue by mejiaj** [Bixal/bx-nextjs-storybook-uswds#29]
## Summary

StorybookJS is currently defaulting to the following rulesets:

- [WCAG 2.0 Level A & AA Rules](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#wcag-20-level-a--aa-rules)
- [WCAG 2.1 Level A & AA Rules](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#wcag-21-level-a--aa-rules)
- [Best Practices Rules](https://github.com/dequelabs/axe-core/blob/develop/doc/rule-descriptions.md#best-practices-rules)

[Source](https://storybook.js.org/docs/writing-tests/accessibility-testing#rulesets) →

## Describe the solution you'd like

For clarity and thorough compliance, we should set it to WCAG 2x AA, Section 508, and Best Practices.

## Alternatives you've considered

n/a

## Additional context

Our current setup is good enough, but adding these rules improves clarity and ensures we're not missing anything.
